src/client/app/translations/data.ts

```diff
@@ -328,8 +328,8 @@ const LocaleTranslationData = {
 		"meter.cumulativeReset": "Cumulative Reset:",
 		"meter.cumulativeResetEnd": "Cumulative Reset End:",
 		"meter.cumulativeResetStart": "Cumulative Reset Start:",
-		"meter.enabled": "Updates:",
-		"meter.endOnlyTime": "End Only Time:",
+		"meter.enabled": "Meter enabled",
```

Comment:
First, the others don't include "Meter". Second, I think the original with "Updates:" is better. Removing the colon was not desired.